zeppelin-web/src/app/spell/spell-base.js

```diff
@@ -31,9 +31,10 @@ export class SpellBase {
    * Consumes text and return `SpellResult`.
    *
    * @param paragraphText {string} which doesn't include magic
+   * @param config {Object}
    * @return {SpellResult}
    */
-  interpret(paragraphText) {
+  interpret(paragraphText, config) {
```

Comment:
I don't have strong preference. But having config as a param is more explicit way to tell to user **"here is config you can use"**